### PR TITLE
ENH: Add scatter plot for Frame

### PR DESCRIPTION
**Pull request by charlesdong1991**
Screen Shot 2019-08-30 at 8 47 22 AM
Screen Shot 2019-08-30 at 8 47 40 AM
Screen Shot 2019-08-30 at 8 47 50 AM

### CODE REVIEW

**Comment by charlesdong1991** [comment]
ideally, should use docstring with is identical to what pandas have, however, i encountered quite annoying linting error issue with docstring, so for now, i keep `s` and `c` docstring very simple. Will change/complete them in future PR where I update docstring for all functions.

File: databricks/koalas/plot.py [hunk 1]
```diff
@@ -509,6 +509,17 @@ def _make_plot(self):
         super(KoalasBarhPlot, self)._make_plot()
 
 
+class KoalasScatterPlot(ScatterPlot, TopNPlot):
+    max_rows = 1000
```
Comment (HyukjinKwon) [inline databricks/koalas/plot.py]: I think we can remove this.

Comment (charlesdong1991) [reply databricks/koalas/plot.py]: ahh, yeah, removed. and added pics in description, i thought i did it yesterday. my bad

**Comment by HyukjinKwon** [comment]
@charlesdong1991, before we merge, can you check if Koalas plot adds "showing top 1,000 elements only" in the output chart when the rows are larger than 1000?

**Comment by charlesdong1991** [comment]
@HyukjinKwon good call! cannot really read it because font size is a bit too small to me 😅 but we probably can tell from density of points that `koalas` plot less.

Screen Shot 2019-08-30 at 9 40 53 AM

**Comment by HyukjinKwon** [comment]
Thanks for testing it out, @charlesdong1991.

**Comment by Ankurneural** [comment]
How to use all the rows for the plot? 
Setting, ks.set_option('compute.max_rows', 50000)
ks.set_option("display.max_rows", 50000) seems not to be working

**Comment by HyukjinKwon** [comment]
`plotting.max_rows` should work for that. There's a documentation here - https://koalas.readthedocs.io/en/latest/user_guide/options.html#available-options

**Comment by Ankurneural** [comment]
> `plotting.max_rows` should work for that. There's a documentation here - https://koalas.readthedocs.io/en/latest/user_guide/options.html#available-options
Thanks! I somehow missed it. It is working as anticipated.

**Comment by Ankurneural** [comment]
@HyukjinKwon I am trying to make subplots using Koalas Dataframe but it comes blank.
d1 is a koalas dataframe.
Sample Code:
fig, (ax1, ax2)  = plt.subplots(1, 2)
ax1 = d1.plot.scatter(x="col1", y="col2")
ax2 = d1.plot.scatter(x="col1", y="col2") 
display(fig)

Any suggestions or help on this?

**Comment by HyukjinKwon** [comment]
@Ankurneural can you create a new ticket with the full codes and output?

**Pull request merged**